microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Get a Microsoft Store billing service by using the Digital Goods API
+description: Use the Digital Goods API to get a Microsoft Store billing service.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: pwa
+ms.date: 11/21/2024
+---
+# Get a Microsoft Store billing service by using the Digital Goods API
+<!-- todo: improve title (in 3+ spots) -->
+<!-- todo: improve the description: field above; sync w/ Sentence 1; develop clear wording for both spots -->
+
+If your Progressive Web App (PWA) is listed in the Microsoft Store, you can provide in-app products and subscriptions by using the Digital Goods API and the Payment Request API.
+
+See also:
+* [Digital Goods API For Microsoft Store PWA Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PwaDigitalGoods/explainer.md)
+* [Payment Request API](https://www.w3.org/TR/payment-request/) at W3C.
+
+
+<!-- ====================================================================== -->
+## Digital Goods API
+
+The Digital Goods API is an interface between your PWA app and Microsoft Store.  The Digital Goods API supports:
+* Querying the details of a digital item from the Microsoft Store backend, such as the item's name, description, and regional price.
+* Consuming or acknowledging purchases.
+* Checking the digital items that are currently owned by the user.
+* Checking the purchase history of the user.
+
+Before making the Digital Goods API available in your app, make sure that - _todo: finish sentence_
```

Comment:
finish sentence "Before making the Digital Goods API available in your app, make sure that"
@runyuany